mw-webapp/src/component/header/Header.tsx

```diff
@@ -162,6 +163,7 @@ export const Header = (props: HeaderProps) => {
       <Link
         className={styles.logo}
         path={pages.home.getPath({})}
+        dataCy={headerAccessIds.items.allWaysLogoBtn}
```

Comment:
it's little bit strange that link is called logoBtn

Reply:
Changed.